node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,9 @@
+const file = process.argv[2];
+console.log('file:', file);
+
+const fs = require('fs');
```

Comment:
This will work fine but it is best to `require` all dependencies at the top of the file